### ISSUE TITLE
Try naive cross entropy loss for retrieval

### ISSUE THREAD

**Issue by yuancu** [happen2me/subgraph-retrieval-toolkit#3]
Currently, I use the contrastive-learning based [`NTXentLoss`](https://kevinmusgrave.github.io/pytorch-metric-learning/losses/#ntxentloss) for calculating the losses. While the original paper used a simpler cross entropy loss. This might contribute to the performance degeneration.

- [x] Change the loss to cross-entropy loss, modify the corresponding similarity calculation code
- [x] Train the model with this loss
